includes/Traits/Publisher.php

```diff
@@ -43,74 +43,18 @@ trait Publisher {
 	 *
 	 * @return string
 	 */
-	public function get_publisher_logo_placeholder() {
+	public function get_publisher_logo_placeholder(): string {
 		return WEBSTORIES_PLUGIN_DIR_URL . 'assets/images/fallback-wordpress-publisher-logo.png';
 	}
 
-	/**
-	 * Returns the publisher data.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @return array Publisher name and logo.
-	 */
-	public function get_publisher_data() : array {
-		$publisher      = $this->get_publisher_name();
-		$publisher_logo = $this->get_publisher_logo();
-
-		return [
-			'name' => $publisher,
-			'logo' => $publisher_logo,
-		];
-	}
-
-	/**
-	 * Get the publisher logo.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @link https://developers.google.com/search/docs/data-types/article#logo-guidelines
-	 * @link https://amp.dev/documentation/components/amp-story/#publisher-logo-src-guidelines
-	 *
-	 * @return string|null Publisher logo image URL, or null if no publisher logos are available.
-	 */
-	public function get_publisher_logo() {
-		$logo_image_url = null;
-
-		$active_publisher_logo = absint( get_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO ) );
-
-		if ( $active_publisher_logo ) {
-			$logo_image_url = wp_get_attachment_image_url( $active_publisher_logo, Image_Sizes::PUBLISHER_LOGO_IMAGE_SIZE );
-
-			if ( ! $logo_image_url ) {
-				$logo_image_url = null;
-			}
-		}
-
-		$placeholder = $this->get_publisher_logo_placeholder();
-
-		/**
-		 * Filters the publisher's logo.
-		 *
-		 * This should point to a square image.
-		 *
-		 * @since 1.0.0
-		 * @since 1.1.0 The $placeholder parameter was deprecated.
-		 *
-		 * @param string|null $logo_image_url URL to the publisher's logo if set.
-		 * @param string      $placeholder    Deprecated.
-		 */
-		return apply_filters( 'web_stories_publisher_logo', $logo_image_url, $placeholder );
```

Comment:
@swissspidy What about the cases where the filter was in use, wouldn't removing this filter be a breaking change? Should the filter be moved to `$sanitizers[ Story_Sanitizer::class ]` then perhaps as a wrapper around the sanitizer? Thoughts?

Reply:
Yeah perhaps, though it would probably make more sense to have a new filter that passes the current story / post object as context.

Reply:
Ok, just want to make sure it wouldn't break for existing usages of the filter -- probably wouldn't be nice to just remove the existing filter unexpectedly.

Reply:
I agree, this filter dont not be removed, it should be maintained.

Reply:
I'll take another look